xlators/protocol/server/src/server-rpc-fops.c

```diff
@@ -147,7 +145,8 @@ server_lookup_cbk(call_frame_t *frame, void *cookie, xlator_t *this,
                  * lookups will be successful, if the lookup
                  * is a soft lookup)
                  */
-                forget_inode_if_no_dentry(state->loc.inode);
```

Comment:
Do we still have use cases for forget_inode_if_no_dentry()?

Reply:
forget_inode_if_no_dentry() is called from server_post_unlink(), server_post_rmdir(), server_post_rename(), server_lookup_cbk(). Not really sure whats the use case when this these cbks gets triggered